docs/manual/en/introduction/Creating-a-scene.html

```diff
@@ -117,47 +101,53 @@ <h2>The result</h2>
 
 		<p>The full code is available below and as an editable [link:https://jsfiddle.net/em4j9hvu/ live example]. Play around with it to get a better understanding of how it works.</p>
```

Comment:
I just realized that we're still advising the users to use three.js from `https://threejs.org/build/three.js` here 😅
The jsfiddle needs to be updated, but maybe in another PR.

Reply:
Ugh! So that's where it came from... 😬

Reply:
This link has already been updated in dev when the builds were temporarily removed.